src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -322,6 +331,168 @@ public void moveToDefaultDirectory() {
         }
     }
 
+    public void updateMoveFileItemText(MenuItem moveFileItem, Path currentFilePath) {
+        FileDirectoryHandler directoryHandler = new FileDirectoryHandler(databaseContext, preferences.getFilePreferences(), dialogService);
+        Optional<FileDirectoryHandler.DirectoryInfo> targetDirectory = directoryHandler.determineTargetDirectory(currentFilePath);
+
+        if (targetDirectory.isPresent()) {
+            FileDirectoryHandler.DirectoryInfo dirInfo = targetDirectory.get();
+            moveFileItem.setText(Localization.lang("Move file to %0", dirInfo.label()));
+        } else {
+            BooleanProperty isMoveFileDisabled = new SimpleBooleanProperty(true);
+            moveFileItem.disableProperty().bind(isMoveFileDisabled);
+            moveFileItem.setText(Localization.lang("Move file"));
```

Comment:
Why are we binding to isMoveFileDisabled here? It is declared locally so its value (true) won't change.

Reply:
Hi @HoussemNasri 
I bind it to follow the consistency in both the classes: LinkedFilesEditor and LinkedFileViewModel
I am aware that we can disable it directly using setDisable().
I will change it. Thanks